openedx_learning/lib/collations.py

```diff
@@ -77,7 +77,7 @@ def db_collation(self, value):
 
         This can be removed when we move to Django 4.2.
         """
-        pass
+        pass  # pylint: disable=unnecessary-pass
```

Comment:
```suggestion
```

If we have a docstring I think is safe to remove the `pass`, right?

Reply:
> If we have a docstring I think is safe to remove the pass, right?

Yes, totally safe. I just left it in because it's a more explicit signal that the class/function is supposed to be blank. But you're right, maybe it's cleaner to just remove it.